tests/__snapshots__/UnifiedSchema.test.ts.snap

```diff
@@ -16,7 +16,7 @@ directive @globalOptions(sourceName: String, endpoint: String, operationHeaders:
 
 directive @httpOperation(path: String, operationSpecificHeaders: ObjMap, httpMethod: HTTPMethod, isBinary: Boolean, requestBaseBody: ObjMap, queryParamArgMap: ObjMap, queryStringOptionsByParam: ObjMap) on FIELD_DEFINITION
 
-type Query @globalOptions(sourceName: "CZIDREST", endpoint: "http://web:3001/") {
```

Comment:
@bchu1 @jfoo1984 should this be here?  I'm not sure why updating the snapshot changed this (also not entirely sure how to get rid of it)

Reply:
update: you have to run `docker compose run gql npm test -- -u`